source/includes/service.md

```diff
@@ -447,7 +449,7 @@ Content-Type: application/json
 "entrypoint": "/usr/sbin/sshd", "image": "tutum/hello-world", 
 "linked_to_service": [{"to_service": "/api/v1/service/80ff1635-2d56-478d-a97f-9b59c720e513/", "name": "db"}], 
 "memory": 2048, "privileged": True, "roles": ["global"], "run_command": "/run.sh", "sequential_deployment": False,
-"tags": [{"name": "tag-1"}], "target_num_containers": 3}
+"tags": [{"name": "tag-1"}], "target_num_containers": 3, "autoredeploy": False}
```

Comment:
Well, no need at all, but I chose to put it as the other ones (optional fields too)